products/customer_analytics/frontend/queries/ZendeskTicketsQuery.tsx

```diff
@@ -0,0 +1,98 @@
+import { Link } from '@posthog/lemon-ui'
+
+import { DataTableNode, NodeKind } from '~/queries/schema/schema-general'
+import { QueryContext, QueryContextColumn } from '~/queries/types'
+import { hogql } from '~/queries/utils'
+
+const DEFAULT_COLUMN_CONFIG = {
+    width: '60px',
+    align: 'center',
+} as QueryContextColumn
```

Comment:
```suggestion
const DEFAULT_COLUMN_CONFIG: QueryContextColumn = {
    width: '60px',
    align: 'center',
}
```

i wouldn't type cast with `as QueryContextColumn` since it bypass type checking